docs/deployment/managers.md

```diff
@@ -1,7 +1,63 @@
 # Platform managers
 
-!!! tip "Under construction"
+## Rules engine
+
+Allows users to execute pre-defined actions based on the data and events in the platform.
+
+These rules are accessible in Settings > Customization > Rules engine
+
+The rules engine is designed to help users automate and streamline their cyber threat intelligence processes.
+
+## History manager
+
+This manager keeps tracks of user/connector interactions on entities in the platform.
+It is designed to help users audit and understand the evolution of their CTI data
```

Comment:
If you want the line break to be made, you need an empty line between the two lines.